### ISSUE TITLE
List of tools and which taxonomies they support

### ISSUE THREAD

**Issue by jfy133** [taxprofiler/taxpasta#2]
GTDB vs NCBI for example? Fix/non-fixed

**Comment by jfy133** [comment]
| Tool | Supported |
|-------|----------------|
| MALT | |
| KRAKEN2 | |
| CENTRIFUGE | |
| METAPHLAN3 | |
| KAIJU | |
| DIAMOND | |
| MOTUS | |

**Comment by Midnighter** [comment]
Since there has been significant progress on https://github.com/nf-core/createtaxdb/, is there any new/additional information to consider @jfy133?

**Comment by jfy133** [comment]
I think just Ganon is missing from the current release of taxprofiler, which supports both NCBI and GTDB - this might be a good summary document to compare against: https://nf-co.re/createtaxdb/1.0.0/docs/usage/usage/faq/#recommended-auxiliary-files

**Comment by jfy133** [comment]
TL;DR, every tool does not really care as long as they are in NCBI-like format; except for tools that do not allowcustom dbs (metaphlan, mOTUs)
